pom.xml

```diff
@@ -18,22 +18,76 @@
             checkstyle.xml
         </maven.checkstyle.plugin.configLocation>
         <java.version>17</java.version>
+        <lombok.mapstruct.binding.version>0.2.0</lombok.mapstruct.binding.version>
+        <mapstruct.version>1.5.5.Final</mapstruct.version>
     </properties>
     <dependencies>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter</artifactId>
         </dependency>
+        <dependency>
```

Comment:
See what changes should be included to this PR in task description.
You can not fix it, but please, pay attention.